### ISSUE TITLE
After update to 2.3.1 auto-type on entry works only once after every start

### ISSUE THREAD

**Issue by wojciech-kopras** [keepassxreboot/keepassxc#1666]
After update to 2.3.1 auto-type on the selected entry action does minimalize of KeepassXC window for auto-typing one for the first auto-type execution. Following executions of auto-type don't minimalize the KeepassXC window. It is "typing" but with KeepassXC window open and focused. After restart of KeepassXC it works again only once.

## Expected Behavior

In earlier versions auto-type action executed on the selected entry normally did this:
1. Minimalized the KeepassXC window
1. Typed the configured sequence of chars (e.g. password) in the window that took focus after minimalizing KeepassXC window.

## Current Behavior

When executed for N-th time (where N >= 2) action auto-type on a selected entry, the window of KeepassXC is not minimalized and chars sequence goes to the KeepassXC window - I can see it becuase the selected entry is changed and opened for edition with the final {ENTER} command. After restart of KeepassXC it works again only once.

## Possible Solution

The window should get minimized after every execution of auto-type action. Maybe the window's state is not registred/read correctly?

## Steps to Reproduce (for bugs)

1. Open a window of a text editor.
1. Open a database with passwords in KeepassXC.
1. Select an entry with some test user/pass data.
1. Execute auto-type action (e.g. with ^V)
1. KeepassXC window gets minimized and login data get written to text editor window.
1. Restore KeepassXC window.
1. Select an entry with some test user/pass data.
1. Execute auto-type action (e.g. with ^V)
1. KeepassXC windows gets not minimized and maybe another entry gets selected and probably opend for editing (depending on the password you have in this entry).

## Context

Now I cannot use auto-type. I need to do copy&paste with passwords.

## Debug Info

KeePassXC - 2.3.1
Revision: 2fcaeea

Libraries:
- Qt 5.2.1
- libgcrypt 1.8.1

Enabled extensions:
- Auto-Type
- Legacy Browser Integration (KeePassHTTP)
- SSH Agent
- YubiKey

These were not on "About..." page:
Operating system: Linux Mint 17.2 Rafaela
CPU architecture: x64
Kernel: 4.4.0-116-generic #140~14.04.1-Ubuntu SMP Fri Feb 16 09:25:20 UTC 2018

**Comment by wojciech-kopras** [comment]
Uups hmm, I have reverted to 2.2.4 version (from https://launchpad.net/~phoerious/+archive/ubuntu/keepassxc/+build/13854735 ) and I have the same problem. So probably sth changed in my system, and it's not a bug, sorry...

**Comment by klaviartur** [comment]
I know, this is 6 years old, but i have the same problem now with version 2.6.6 and would be very pleased to know if and how you were able to solve it then, if you even remember.

Thx

**Comment by phoerious** [comment]
2.6.6 has been EOL for a long time. Please upgrade to 2.7.7.

**Comment by klaviartur** [comment]
oh, thx for the hint.
I was relying on my package manager to keep it updated automatically and didn't even check for a newer version manually. 
Shame on me!
I'll check why apt doesn't update it.

**Comment by phoerious** [comment]
You may have to add our PPA.

**Comment by klaviartur** [comment]
I'm on it already. Thx.

**Comment by klaviartur** [comment]
Unfortunately the problem persists even with 2.7.7
I'll open a new issue with a better description. I think I pinned it down to a specific case.

**Comment by droidmonkey** [comment]
@klaviartur this issue was fixed already for pending 2.7.8

**Comment by klaviartur** [comment]
well, then i just threw out a ton of time finishing that bug report :D
thx for the info.
